integrations/integration-vip-config.php

```diff
@@ -109,32 +109,24 @@ protected function get_vip_config_from_file( string $slug ) {
 	 * @private
 	 */
 	public function is_active_via_vip(): bool {
-		// Return false if blocked on org.
-		if ( $this->get_value_from_config( 'org', 'status' ) === Org_Integration_Status::BLOCKED ) {
-			return false;
-		}
-
-		$env_status = $this->get_value_from_config( 'env', 'status' );
-
-		// Return false if blocked on env.
-		if ( Env_Integration_Status::BLOCKED === $env_status ) {
-			return false;
-		}
-
-		// Look into network_sites config before because if not present we will fallback to env config.
-		$network_site_status = $this->get_value_from_config( 'network_sites', 'status' );
-
-		if ( Env_Integration_Status::ENABLED === $network_site_status ) {
-			return true;
-		}
```

Comment:
I noticed that this function can be simplified via refactoring so added `get_site_status()` function which will also help us in future by giving the actual status of the integration instead of just boolean.